documentation/modules/exploit/linux/http/roxy_wi_exec.md

```diff
@@ -30,62 +27,49 @@ chown -R www-data:www-data haproxy-wi
 9. Do: `run`
 10. You should get a shell as the user running the Roxy-WI server.
 
+## Targets
+
+### 0
+
+This executes a Unix command.
+
+### 1
+
+This uses a Linux dropper to execute code.
+
 ## Options
 
 ### TARGETURI
-The base path to Roxy-WI. The default value is `/`
+
+Set `TARGETURI` if the Roxy-WI is installed at a custom path.
```

Comment:
So your explaining what TARGETURI does by.....asking the user to refer to the definition of TARGETURI here again? This doesn't make any sense as now you have a definition which points to itself as the explanation. You previous text was fine, this new one is just confusing.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16793/commits/72b1dbfeee3a7aa69916b427eeed6f73a85f4a9a